public/app/features/alerting/unified/components/receivers/ReceiversTable.tsx

```diff
@@ -49,22 +103,107 @@ function ReceiverHealth({ errorsByReceiver }: ReceiverHealthProps) {
   );
 }
 
-interface Props {
-  config: AlertManagerCortexConfig;
-  alertManagerName: string;
-}
-
 const useContactPointsState = (alertManagerName: string) => {
   const contactPointsStateRequest = useUnifiedAlertingSelector((state) => state.contactPointsState);
   const { result: contactPointsState } = (alertManagerName && contactPointsStateRequest) || initialAsyncRequestState;
-  const receivers = contactPointsState?.receivers ?? {};
+  const receivers: ReceiversState = contactPointsState?.receivers ?? {};
   const errorStateAvailable = Object.keys(receivers).length > 0; // this logic can change depending on how we implement this in the BE
   return { contactPointsState, errorStateAvailable };
 };
 
+interface ReceiverItem {
+  name: string;
+  types: string[];
+  provisioned?: boolean;
+}
+
+interface IntegrationItem {
```

Comment:
nit: we sometimes call these `notifiers`, perhaps we should rename this to `NotifierStatus` or something similar?

Reply:
💯